deployment/helm/skaha/skaha-config/launch-carta.yaml

```diff
@@ -27,7 +26,7 @@ spec:
       automountServiceAccountToken: false
       enableServiceLinks: false
       restartPolicy: OnFailure
-      ${skaha.schedulegpu}
+      {{ template "skaha.job.nodeAffinity" . }}
```

Comment:
What happens to this if the deployer doesn't add an entry to their values.yaml file?  Does skaha (at runtime) simply remove it?

Reply:
Here is the implementation of the `skaha.job.nodeAffinity` template function:
```
{{/*
The affinity for Jobs.  This will import the YAML as defined by the user in the deployment.skaha.sessions.nodeAffinity stanza.
*/}}
{{- define "skaha.job.nodeAffinity" -}}
{{- with .Values.deployment.skaha.sessions.nodeAffinity }}
      affinity:
        nodeAffinity:
{{ . | toYaml | indent 10 }}
{{- end }}
{{- end }}
```

The `with` command prevents further access if its argument is missing.  If the `nodeAffinity` is absent, it is ignored.